control/trajectory_follower/obstacle_collision_checker/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>obstacle_collision_checker</name>
+  <version>0.1.0</version>
+  <description>The obstacle_collision_checker package</description>
+  <author email="satoshi.tanaka@tier4.jp">Satoshi Tanaka</author>
+  <maintainer email="kenji.miyake@tier4.jp">Kenji Miyake</maintainer>
+  <license>Apache 2</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>autoware_planning_msgs</depend>
+  <depend>autoware_utils</depend>
+  <depend>vehicle_info_util</depend>
+  <depend>boost</depend>
+  <depend>diagnostic_updater</depend>
+  <depend>eigen</depend>
+  <depend>geometry_msgs</depend>
+  <depend>pcl_ros</depend>
+  <depend>rclcpp</depend>
+  <depend>std_msgs</depend>
+  <depend>tf2</depend>
+  <depend>tf2_eigen</depend>
+  <depend>tf2_geometry_msgs</depend>
+  <depend>tf2_ros</depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
Replace ament_cmake_cppcheck with ament_lint_cpmmon, and fix lint.

Reply:
Now I understand why linter doesn't work.